doi/src/main/webapp/paths/doi-instance.yaml

```diff
@@ -0,0 +1,46 @@
+post:
+  tags:
+    - "instances"
+  summary: "Initialize a new DOI"
+  description: "Generates a VOSpace directory the calling user will have permission to upload DOI data to, and an XML file using DataCite metadata XSD format."
+  operationId: "createDOI"
+  requestBody:
+    required: true
+    content:
+      application/json:
+        schema:
+          type: "string"
```

Comment:
I like how the JSON schema in defined in doi-instance-manage.yaml. I'd updated all JSON schema references to that syntax.